library/globals.inc.php

```diff
@@ -938,13 +938,55 @@
       xl('This will use the custom Statment showing the description instead of the codes.')
     ),
 	 
+    'show_aging_on_custom_statement' => array(
+      xl('Show Aging on Custom Statement'),
+      'bool',                           // data type
+      '0',                              // default = false
+      xl('This will Show Aging on the custom Statment.')
+    ),
+
+    'use_statement_print_exclusion' => array(
+      xl('Exclude Statements from Printing'),
+      'bool',                           // data type
+      '0',                              // default = false
+      xl('This will use the Following settings to Exclude Patient Statements from Printing.')
```

Comment:
I will submit 2 more exclusions one for an insurance company and one for a patient( will add a field to the "Choices" tab in demographics. It will be a little bit before I can work on the next two.